examples/howto/data_container.myst.md

```diff
@@ -47,7 +47,7 @@ az.style.use("arviz-darkgrid")
 
 After building the statistical model of your dreams, you're going to need to feed it some data. Data is typically introduced to a PyMC model in one of two ways. Some data is used as an exogenous input, called `X` in linear regression models, where `mu = X @ beta`. Other data are "observed" examples of the endogenous outputs of your model, called `y` in regression models, and is used as input to the likelihood function implied by your model. These data, either exogenous or endogenous, can be included in your model as wide variety of datatypes, including numpy `ndarrays`, pandas `Series` and `DataFrame`, and even pytensor `TensorVariables`. 
 
-Although you can pass these "raw" datatypes to your PyMC model, the best way to introduce data into your model is to use one of two {func}`pymc.data.Data` containers. These containers make it extremely easy to work with data in a PyMC model. They offer a range of benefits, including:
+Although you can pass these "raw" datatypes to your PyMC model, the best way to introduce data into your model is to use one of two {func}`pymc.Data` containers. These containers make it extremely easy to work with data in a PyMC model. They offer a range of benefits, including:
```

Comment:
That will work. For future reference, the id is the title at the page: https://www.pymc.io/projects/docs/en/latest/api/generated/pymc.Data.html. This does look like a class, but classes are explicited in the docs, for example: https://www.pymc.io/projects/docs/en/latest/api/distributions/generated/pymc.Beta.html.

In the old 3.x days it was different, but now everything is documented at their "recommended import path" as users don't really care that Data is inside the data.py file, only that they can use it as pymc.Data

Reply:
It would be nice if reference validity was checked as a pre-commit hook

Reply:
This can't be really checked in pre-commit. The docs need to be built to see all the targets that exists and which ones can't be resolved. Ideally, we would have a sphinx build process without warnings so that we could tell readthedocs to fail CI if sphinx prints any warning (like a cross-reference not working). There are currently many warnings (though not as many as in main pymc repo) so we can't do that afaik